docs/development/variables/environment-variables.mdx

```diff
@@ -107,6 +107,7 @@ These are the environment variables used in Mage.
 | `MAGE_PUBLIC_HOST`               | The public host url that can be used to access the Mage app. This value will be used in emails or other notifications.                                          | `http://localhost:6789`                                                  |
 | `MAX_PRINT_OUTPUT_LINES`         | The max number of `stdout` lines, such as from `print` statements, included in a message from the websocket server. Defaults to `1000`.                         | `100`                                                                    |
 | `REQUIRE_USER_AUTHENTICATION`    | Enable user authentication in Mage. [More information](/production/authentication/overview)                                                                     | 1                                                                        |
+| `RUN_PIPELINE_IN_ONE_PROCESS`    | When True, pipelines will run in one process, unless otherwise specified in the settings for a given pipeline.
```

Comment:
Provide an example value for this setting in the "Example values" field of this table, e.g., 1, to be consistent with other boolean settings.